include/mbedtls/ecp.h

```diff
@@ -61,6 +61,26 @@
 
 #define MBEDTLS_ERR_ECP_IN_PROGRESS                       -0x4B00  /**< Operation in progress, call again with the same parameters to continue. */
 
+/* Flags indicating whether to include code that is specific to certain
+ * types of curves. These flags are for internal library use only. */
```

Comment:
The only point is to avoid defining the `mbedtls_ecp_mul_add` functions, which was [preferred in an old discussion](https://github.com/ARMmbed/mbedtls/pull/2013#discussion_r217111400).

Reply:
ok but this has the drawback to expose in the public interface symbols that are not supposed to be exposed. In the old discussion I don't see this drawback being mentioned. @mpg, I can see you were part of the discussion mentioned above, what do you think about this public vs internal symbol issue?

Reply:
I think I'd rather not worry too much about that for now, we're not even sure how much of `ecp.h` is still going to be public in 3.0, but hopefully `mbedtls_ecp_mul_add()` will no longer be, so we can clean it up at this point.

Reply:
ok, make sense, fine by me then.